assets/messages.yml

```diff
@@ -796,7 +796,7 @@ messages:
         *Thank you for your report!*
         However, this issue is {color:#FF5722}*Invalid*{color}.
 
-        The server you are playing on is modified.
+        Your game or server is modified.
```

Comment:
Ah, I did not notice that Java uses a similar message. Feel free to change that message as well. But in both cases only change it if you think it would improve the messages. No worries if you don't want to make these changes.